relevanceai/datasets.py

```diff
@@ -480,6 +473,13 @@ def get_machine_learning_research_dataset():
     raise NotImplementedError
 
 
-get_dummy_ecommerce_dataset = get_ecommerce_1_dataset
-get_sample_ecommerce_dataset = get_ecommerce_2_dataset
-get_ecommerce_dataset = get_ecommerce_3_dataset
+### For backwards compatability
+
+get_ecommerce_1_dataset = get_ecommerce_dataset_encoded

```

Comment:
I think we should make full use of Python's multiple assignment property as per the comment as I don't see a need to assign it again as it reduces maintainability in future as we are less sure about what's equal.

Reply:
Ahh I understand your meaning of course!